frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultViews/DialogUtils/QuizResultDetailsDialog.scss

```diff
@@ -0,0 +1,147 @@
+@import "../../../../../../../../styles/styles";
+
+
+.QuizResultDetailsDialog {
+
+    .appBar {
+        position: fixed;
+    }
+
+    .title {
+        font-size: 24px;
+        font-weight: 600;
+        padding: 0 $spacing-05;
+    }
+
+    .container {
+        height: calc(100% - 3*#{$spacing-09});
```

Comment:
why do you need to do the `calc` in so many places? Seems like a hack and not a proper solution